xwiki-platform-core/xwiki-platform-oldcore/src/main/java/com/xpn/xwiki/user/impl/xwiki/MyBasicAuthenticator.java

```diff
@@ -35,10 +35,21 @@
 
 import com.xpn.xwiki.XWikiContext;
 import com.xpn.xwiki.XWikiException;
+import com.xpn.xwiki.internal.user.UserAuthenticatedEventNotifier;
 import com.xpn.xwiki.web.Utils;
 
 public class MyBasicAuthenticator extends BasicAuthenticator implements XWikiAuthenticator
 {
+
+    private UserAuthenticatedEventNotifier userAuthenticatedEventNotifier;
+
+    private UserAuthenticatedEventNotifier getUserAuthenticatedEventNotifier() {
```

Comment:
This is not following XWiki codestyle.

Reply:
Is this not intended to be reviewed by checkstyle at build time ?

Btw I'll fix it.

Reply:
MyBasicAuthenticator is skipped in the checkstyle configuration as far as I can see.

But anyway relying on the build for that is not very effective, you should configure your IDE formatter and use it instead so that you don't have to worry about that at build time.

Reply:
Yes of course, I need to dig deeper in IntelliJ settings.

Reply:
See https://dev.xwiki.org/xwiki/bin/view/Community/CodeStyle/JavaCodeStyle/#HIntelliJIDEA